app/helpers/attachments_helper.rb

```diff
@@ -60,7 +60,9 @@ def attachment_component_params(attachment, alternative_format_contact_email: ni
   end
 
   def block_attachments(attachments = [], alternative_format_contact_email = nil)
-    attachments.map do |attachment|
+    attachments
```

Comment:
could make the calling code use `renderer.block_attachments(item.attachments_ready_for_publishing...)`

Reply:
As discussed, this is better for now, as we the alternative would force us to write excessive tests or refactor the abstractions in Presenters to better represent the requirements - not the right time for that.